client/index.pug

```diff
@@ -5,8 +5,5 @@ html
 		body
 			#root
 
-			each cssFile in htmlWebpackPlugin.files.css
-				style !{compilation.assets[cssFile.substr(htmlWebpackPlugin.files.puglicPath.length)].source()}
-
 			each jsFile in htmlWebpackPlugin.files.js
-				script !{compilation.assets[jsFile.substr(htmlWebpackPlugin.files.publicPath.length)].source()}
\ No newline at end of file
+				script !{compilation.assets[jsFile.substr(htmlWebpackPlugin.files.publicPath.length)].source()}
```

Comment:
What's different here? New line vs no new line?

Reply:
Yep, files should end in an empty newline